private/model/cli/gen-api/main.go

```diff
@@ -180,7 +182,10 @@ func writeServiceFiles(g *generateInfo, pkgDir string) {
 	Must(writeExamplesFile(g))
 
 	if g.API.HasEventStream {
-		Must(writeAPIEventStreamTestFile(g))
+		// has stream APIs with host prefix, which our tests break on, skip codegen for now
+		if g.API.PackageName() != "cloudwatchlogs" {
```

Comment:
nit: why not combine 2 `if` judgement together like this 
`if g.API.HasEventStream && g.API.PackageName() != "cloudwatchlogs"`

Reply:
at first glance we should not hard code the `cloudwatchlogs` and its tests, but this is the best solution up to now

Reply:
Mostly just to isolate the cloudwatch check so it's minimally less effort to rip out if we ever fix this.